iam_policy/templates/keycloak_ecs_task_role_policy.json.tpl

```diff
@@ -1,20 +1,27 @@
 {
   "Version": "2012-10-17",
-    "Statement": [
-      {
-        "Effect": "Allow",
-         "Action": [
-           "sns:Publish"
-         ],
-         "Resource": "arn:aws:sns:eu-west-2:${account_id}:tdr-notifications-${environment}"
-      },
-      {
-        "Effect": "Allow",
-        "Action": [
-          "kms:Decrypt",
-          "kms:GenerateDataKey"
-        ],
-        "Resource": "${kms_arn}"
-      }
-    ]
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "sns:Publish",
+        "rds-db:connect",
+        "kms:Decrypt",
+        "kms:GenerateDataKey"
+      ],
+      "Resource": [
+        "${kms_arn}",
+        "arn:aws:rds-db:eu-west-2:${account_id}:dbuser:${cluster_resource_id}/keycloak_user",
+        "arn:aws:sns:eu-west-2:${account_id}:tdr-notifications-${environment}"
+      ]
+    },
+    {
```

Comment:
Is this still needed, as it appears these permissions are in the first statement of the policy?

Reply:
Ah good spot, I was trying to merge them and forgot to remove the originals.